tests/js/realm-tests.js

```diff
@@ -117,6 +117,16 @@ module.exports = BaseTest.extend({
         TestCase.assertEqual(Realm.defaultPath, newPath, "defaultPath should have been updated");
     },
 
+    testRealmSchemaVersion: function() {
+        TestCase.assertEqual(Realm.schemaVersion(Realm.defaultPath), 0xFFFFFFFFFFFFFFFF);
```

Comment:
I didn't exactly know what you meant especially given your sentence about compatibility. :smile: I don't think this API means anything for Realms being compatible across bindings.

Each binding should make sense for its given language. That is why the Objective-C API returns a constant (`RLMNotVersioned`) and the Swift API returns `nil`. Each of those makes sense for their respective languages. For JS, I'd say the appropriate value is definitely `-1`, but `null` is also a possibility, though I'd say it's less good.